docs/section19/19-19.md

```diff
@@ -0,0 +1,96 @@
+            - Get list with the hvac system's fan system's supply, return, exhaust and relief fan CFM: `fan_sys_CFM = list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(B_RMI,fan_sys_b)["Fan_Power"]["Fan_CFM"])` 
+                    
+            - Get the hvac system total supply fan kW, need to get the first element in the list of fan powers:`hvac_sys_total_supply_fan_power_b = fan_sys_kW[0]` 
+            - Get the hvac system total return fan kW, need to get the second element in the list of fan powers:`hvac_sys_total_return_fan_power_b = fan_sys_kW[1]` 
+            - Get the hvac system total exhaust fan kW, need to get the third element in the list of fan powers:`hvac_sys_total_exhaust_fan_power_b = fan_sys_kW[2]` 
+            - Get the hvac system total relief fan kW, need to get the fourth element in the list of fan powers:`hvac_sys_total_relief_fan_power_b = fan_sys_kW[3]`     
+
+            - Calculate the total modeled fan power for the hvac system in the baseline: `total_fan_power = hvac_sys_total_supply_fan_power_b + hvac_sys_total_return_fan_power_b + hvac_sys_total_exhaust_fan_power_b + hvac_sys_total_relief_fan_power_b`  
+            - Get the total supply cfm, need to get the first element in the list of fan CFMs: `supply_cfm_b = fan_sys_CFM[0]` 
+            - Get the total exhaust cfm, need to get the third element in the list of fan CFMs: `total_exhaust_cfm = fan_sys_CFM[2]`   
+            - Get the total relief cfm, need to get the four element in the list of fan CFMs: `total_relief_cfm = fan_sys_CFM[3]`  
+
+            - Cycle through each zone associated with the HVAC system to get the exhaust fan power: `for zone_b in zone_list:`  
+                - Create a zone object from the zone id (not sure how the RCT will do this): `zone_b_obj = zone_b`  
+                - Check if there are any exhaust fan objects associated with the zone: `if zone_b_obj.zonal_exhaust_fan != Null:`  
+                    - Get the fan power: `fan_elec_power = get_fan_object_electric_power(B_RMI,zone_b_obj.zonal_exhaust_fan)`    
+                    - Add to the total power: `total_fan_power = total_fan_power + fan_elec_power`                    
+            
+            Below determines the expected fan power:
+            - Check if there are any energy recovery objects associated with the fan system object: `if fan_sys_b.air_energy_recovery != Null:` 
+                - Get the enthalpy recovery ratio: `enthalpy_rec_ratio = fan_sys_b.air_energy_recovery.enthalpy_recovery_ratio`  
+                - Assume that the outdoor air flow through the ERV is equal to the minimum outdoor airflow: `ERV_OA_air_flow = fan_sys_b.minimum_outdoor_airflow`  
+                - Assume that the exhaust air flow through the ERV is equal to the maximum of either the total exhaust cfm or total relief cfm associated with the fan system: `ERV_Ex_air_flow = max(total_exhaust_cfm,total_relief_cfm)`  
```

Comment:
I don't think we should include relief CFM, just the exhaust CFM.

Reply:
@mkarpman terminal fans are covered by G3.1.3.14. Maybe we should add a rule for this and check that they are modeled correctly for 6 and 8 and that no fan power is modeled for the other baseline systems?

Reply:
Outcome, ignore terminal fans 2/16/2023 because checked by function that does HVAC system mapping.

Reply:
Implemented.